### PR TITLE
Bug 1906441:  fix keys for HorizontalNav and NavBar

### PR DESCRIPTION
**Pull request by rhamilto**

### CODE REVIEW

File: frontend/public/components/utils/horizontal-nav.tsx [hunk 1]
```diff
@@ -179,10 +179,10 @@ export const NavBar = withRouter<NavBarProps>(({ pages, baseURL, basePath }) =>
           'co-m-horizontal-nav-item--active': matchURL?.isExact,
         });
         return (
-          <li className={klass} key={name}>
+          <li className={klass} key={nameKey ? nameKey : name}>
```
Comment (spadgett) [inline frontend/public/components/utils/horizontal-nav.tsx:182]:
```suggestion
          <li className={klass} key={nameKey || name}>
```

Comment (rhamilto) [reply frontend/public/components/utils/horizontal-nav.tsx]: Updated.  Thanks, @spadgett

**Review by spadgett — APPROVED** [review]
/lgtm

Thanks!

**Pull request merged**